### ISSUE TITLE
Ignore case

### ISSUE THREAD

**Issue by mcandre** [h/find-file-in-repository#16]
Would be great to show results regardless of upper/lower casing, for easier typing!

**Comment by h** [comment]
@mcandre Yes, this sounds like a useful change -- if you'd like to turn this into a PR, I'd be happy to merge it in

**Comment by Sushzhhz** [comment]
```c#

``` **

**Comment by Sushzhhz** [comment]
Workaround example with use-package:

(setq read-file-name-completion-ignore-case t)
(ido-mode)

(use-package find-file-in-repository
  :bind ("C-x f" . find-file-in-repository))
